src/main/java/com/bookappstore/model/User.java

```diff
@@ -30,6 +40,48 @@ public class User {
     @Column(nullable = false)
     private String lastName;
     private String shippingAddress;
+    @ManyToMany(fetch = FetchType.LAZY)
+    @JoinTable(name = "users_roles",
+            joinColumns = @JoinColumn(name = "user_id"),
+            inverseJoinColumns = @JoinColumn(name = "role_id"))
+    private Set<Role> roles;
```

Comment:
It's better to initialise when you declare
```suggestion
    private Set<Role> roles = new HashSet<>();
```

Reply:
good tip, my bad